internal/report/prometheus.go

```diff
@@ -9,22 +9,33 @@ const namespace = "popeye"
 
 // Metrics
 var (
-	score = prometheus.NewGauge(prometheus.GaugeOpts{
+	score = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Namespace: namespace,
 		Name:      "score",
 		Help:      "Score of kubernetes cluster.",
-	})
-	grade = prometheus.NewGauge(prometheus.GaugeOpts{
+	},
+		[]string{
+			"cluster",
+			"namespace",
+		})
+	grade = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
@eminugurkenar I think we should track the grade with the score as stipulated above.

Reply:
removed grade metric , it got merged to score. it is much better presentation now i think.